.github/workflows/build-checker.yaml

```diff
@@ -23,9 +23,37 @@ jobs:
       
       - name: Install dependencies
         run: npm ci --ignore-scripts
-
+      
       - name: Folderslint
         run: npm run folderslint
+  
+  LintStagedFiles:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      
+      - name: Get changed files in the docs folder
```

Comment:
In the docs folder?

Reply:
Yeah - that should be subgraphs. Also, made another update to only grab changed files in the subgraphs folder.